### PR TITLE
Update index.en-US.md to document tagRender

### PR DESCRIPTION
**Pull request by georgeciobanu**
tagRender is currently available but not documented, so users do not know to use it.

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link
[Original bug and conversation](https://github.com/ant-design/ant-design/issues/42228#issuecomment-1542413981)

### 📝 Changelog
Simply adding a line of documentation for the existing Cascader API function tagRender.

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at d6c9b38</samp>

Added `tagRender` property to Cascader component API. This property enables custom rendering of tags for multiple selection.

### <samp>🤖 Generated by Copilot at d6c9b38</samp>

* Add `tagRender` property to Cascader component to customize tags in multiple mode ([link](https://github.com/ant-design/ant-design/pull/42262/files?diff=unified&w=0#diff-5ced7e6e1259091e502d7b0512332fdeb05319823a37c5ae3fa7ffb1ccf8aa50R58))

### CODE REVIEW

**Comment by georgeciobanu** [comment]
@zombieJ followed up on your suggestion in https://github.com/ant-design/ant-design/issues/42228#issuecomment-1542413981

Hope this helps and thank you again for the amazing work you are doing!

File: components/cascader/index.en-US.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ Cascade selection box.
 | defaultValue | Initial selected value | string\[] \| number\[] | \[] |  |
 | disabled | Whether disabled select | boolean | false |  |
 | displayRender | The render function of displaying selected options | (label, selectedOptions) => ReactNode | label => label.join(`/`) | `multiple`: 4.18.0 |
+| tagRender | Custom render function for tags in `multiple` mode | (label: string, onClose: function, value: string) => ReactNode | - |  |
```
Comment (zombieJ) [inline components/cascader/index.en-US.md:58]: TS def not correct, should be `({ label: string, onClose: function, value: string }) => ReactNode`.

And CN version:
| tagRender | 自定义 tag 内容 render，仅在多选时生效 | ({ label: string, onClose: function, value: string }) => ReactNode | - |  |
```txt

```

Comment (li-jia-nan) [reply components/cascader/index.en-US.md:58]: 这老哥现在凌晨三点，要么先合进去，中文我来补充一下？

Comment (li-jia-nan) [reply components/cascader/index.en-US.md:58]: - #42303

**Review by li-jia-nan — APPROVED** [review]

**Comment by georgeciobanu** [comment]
Thank you for finishing this and apologies for being a bit slow to react to your wonderful feedback!

It looks like y'all did everything, let me know if you want me to do anything else.

**Pull request merged**